packetbeat/protos/cassandra/frame.go

```diff
@@ -0,0 +1,900 @@
+/**
+https://github.com/apache/cassandra/blob/trunk/doc/native_protocol_v4.spec
+
+  The CQL binary protocol is a frame based protocol. Frames are defined as:
+
+      0         8        16        24        32         40
+      +---------+---------+---------+---------+---------+
+      | version |  flags  |      stream       | opcode  |
+      +---------+---------+---------+---------+---------+
+      |                length                 |
+      +---------+---------+---------+---------+
+      |                                       |
+      .            ...  body ...              .
+      .                                       .
+      .                                       .
+      +----------------------------------------
+
+  The protocol is big-endian (network byte order).
+
+  some code derived from https://github.com/gocql/gocql
```

Comment:
makes me wonder if gocql could be used directly instead of copying and modifying code.